### PR TITLE
Summit 2016

### PR DESCRIPTION
**Pull request by tjsavage**
Adding starter site for Summit 2016 announcement. Moved 2015 summit site to /summit-2015

### CODE REVIEW

**Comment by tjsavage** [comment]
@arthurevans could you take a look/let me know if I messed anything up?

File: app/summit-2016/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+    }
+    .misc-banner {
+        background: url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Sm@2x.png) center center, url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Md@2x.png) center center, url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Lg@2x.png) center center #1F1F35;
+        background-size: 420px 210px, 600px 300px, 600px 600px
+    }
+}
+@media (max-width:800px) and (-webkit-min-device-pixel-ratio:2) and (min-resolution:192dpi) {
+    .wait-list-banner {
+        background: url(../images/Pink_BG_Pattern/Mobile/Summit_PinkBGMobile@2x.png) center center #ED5185;
+        background-size: cover
```
Comment (notwaldorf) [inline app/summit-2016/styles/main.css]: needs a ;

File: app/summit-2016/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+        background: url(../images/Angle_Pattern/Lt_Blue_BG/Summit_AnglePattern_Blue_Sm@2x.png) center center, url(../images/Angle_Pattern/Lt_Blue_BG/Summit_AnglePattern_Blue_Lg@2x.png) center center #147BCC;
+        background-size: 630px 305px, 512px 300px
+    }
+    .wait-list-banner {
+        background: url(../images/Pink_BG_Pattern/Desktop/Summit_PinkBGPattern_Left@2x.png) center left repeat-y, url(../images/Pink_BG_Pattern/Desktop/Summit_PinkBGPattern_Right@2x.png) center right repeat-y #ED5185;
+        background-size: 25%
+    }
+    .misc-banner {
+        background: url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Sm@2x.png) center center, url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Md@2x.png) center center, url(../images/Angle_Pattern/DkBlue_Royal_Footer/Summit_AnglePattern_Lg@2x.png) center center #1F1F35;
+        background-size: 420px 210px, 600px 300px, 600px 600px
```
Comment (notwaldorf) [inline app/summit-2016/styles/main.css]: oh i see nothing has ;

**Comment by notwaldorf** [comment]
I think this LGTM. It's terrifying. I say ship it and put me in the doghouse if it breaks, i guess?

File: app/summit-2016/index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+                        </div>
+                    </div>
+                    <a class="london-bg" aria-label="Event map" target="_blank" href="https://www.google.com/maps/place/Tobacco+Dock/@51.5079623,-0.0610297,17z/data=!3m1!4b1!4m13!1m7!3m6!1s0x48760331dd10ff3f:0x438a8f1c8d683e45!2sTobacco+Dock!3b1!8m2!3d51.507959!4d-0.058841!3m4!1s0x48760331dd10ff3f:0x438a8f1c8d683e45!8m2!3d51.507959!4d-0.058841"></a>
+                    <div class="london-info responsive horizontal layout">
+                        <div class="about-waterfront flex vertical center layout">
+                            <section>
+                                <h3>Tobacco Dock. London.</h3>
+                                <p>Located in the heart of London's enterprise zone and walking distance from the Tower of London, Tobacco Dock will be home to the second Polymer Summit. Built in the early 1800's and once home to a warehouse for imported Tobacco, the venue is now a thriving event space featuring striking timber beams, industrial ironwork, and brick courtyards.</p>
```
Comment (arthurevans) [inline app/summit-2016/index.html]: "imported Tobacco" => "imported tobacco"

File: app/summit-2016/index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                    <div class="london-scrim"></div>
+                    <div class="badge-white"></div>
+                </a>
+                <footer>
+                    <div class="footer-inner responsive horizontal center layout">
+                        <a class="horizontal center layout" target="_blank" href="https://www.polymer-project.org">
+                            <div class="logo"></div>
+                            <div class="title">Polymer</div>
+                        </a>
+                        <div class="flex"></div><a target="_blank" href="http://polymer.github.io/LICENSE.txt" data-outbound="license-txt">&copy; 2015 Polymer Authors</a>
```
Comment (arthurevans) [inline app/summit-2016/index.html]: Copyright 2016

**Comment by arthurevans** [comment]
LGTM with minor nits. I'm just going to merge and tweak them I think.

**Comment by abdonrd** [comment]
The Polymer Summit website has its own repository?

**Comment by ebidel** [comment]
Part of this one: https://github.com/Polymer/docs/tree/master/app/summit-2016

**Comment by abdonrd** [comment]
@ebidel but that is a bundled version. I meant the original project.

**Comment by ebidel** [comment]
Not sure what you mean. There's no standalone repo for the summit site.

**Comment by abdonrd** [comment]
@ebidel in the [summit-2016/src/](https://github.com/Polymer/docs/tree/master/app/summit-2016/src) there is only a bundled [`elements.html`](https://github.com/Polymer/docs/blob/master/app/summit-2016/src/elements.html). I can't contribute in a bundled version. I was asking about the unbundled version.

**Comment by ebidel** [comment]
Gotcha. @tjsavage might be able to point us to how he generates it. It looks like 2015 is also a compiled bundle.

**Comment by notwaldorf** [comment]
For 2015 I think I got the bundle from @robdodson (and didn't generate it myself), because it was the easiest way to bring it over and we were in a pre-IO rush.

**Comment by ebidel** [comment]
I think we lost https://github.com/Polymer/docs/tree/docs-05/summit somewhere in the mix.

**Comment by arthurevans** [comment]
So, Rob only checked in the built version of the summit site when we migrated to the new site. And I think Taylor just cloned that version to create a new version of the summit site. Not sure what to do if we need to update some of the infrastructure. What were you looking to change @abdonrd.

**Comment by abdonrd** [comment]
@arthurevans nothing super important. But the favicon for example.

**Comment by abdonrd** [comment]
@arthurevans @tjsavage Right now share the Summit site in social networks is a little poor. I would like to propose add Open Graph meta tags to the Summit site.

Example: [abdonrd/abdonrd.com/index.html#L45-L61](https://github.com/abdonrd/abdonrd.com/blob/develop/index.html#L45-L61)

Card validator: https://cards-dev.twitter.com/validator
Facebook Sharing Debugger: https://developers.facebook.com/tools/debug/

**Pull request merged**